readthedocs/projects/signals.py

```diff
@@ -2,12 +2,12 @@
 
 import django.dispatch
 
-before_vcs = django.dispatch.Signal(providing_args=['version', 'environmemt'])
+before_vcs = django.dispatch.Signal()
 
-before_build = django.dispatch.Signal(providing_args=['version', 'environmemt'])
-after_build = django.dispatch.Signal(providing_args=['version'])
+before_build = django.dispatch.Signal()
+after_build = django.dispatch.Signal()
 
-project_import = django.dispatch.Signal(providing_args=['project'])
+project_import = django.dispatch.Signal()
 
 # Used to purge files from the CDN
-files_changed = django.dispatch.Signal(providing_args=['project', 'version'])
+files_changed = django.dispatch.Signal()
```

Comment:
I removed it because it triggered a deprecation warning saying that it will be removed in Django 4.0 and it has no effect other than as documentation for arguments received, but it was not used at all.